tutorials/physics/interpolation/advanced_physics_interpolation.rst

```diff
@@ -0,0 +1,119 @@
+.. _doc_advanced_physics_interpolation:
+
+Advanced Physics Interpolation
+==============================
+
+Although the previous instructions will give satisfactory results in a lot of games, in some cases you will want to go a stage further to get the best possible results and the smoothest possible experience.
+
+Exceptions to automatic physics interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Even with physics interpolation active, there may be some local situations where you would benefit from disabling automatic interpolation for a :ref:`Node<class_Node>` (or branch of the :ref:`SceneTree<class_SceneTree>`), and have the finer control of performing interpolation manually.
+
+This is possible using the :ref:`Node.set_physics_interpolated<class_Node_method_set_physics_interpolated>` function which is present in all Nodes. If you for example, set this interpolated flag to false for a Node, the children will recursively also be affected. This means you can easily disable interpolation for an entire subscene.
+
+The most common situation where you may want to perform your own interpolation is Cameras.
+
+Cameras
+^^^^^^^
+
+In many cases a :ref:`Camera<class_Camera>` can just use automatic interpolation just like any other node. However, for best results, especially at low physics tick rates, it is recommended that you take a manual approach to Camera interpolation.
+
+This is because viewers are very sensitive to Camera movement. A Camera that realigns slightly every, say 1/10th of a second at 10tps tick rate will often be noticeable, and you can get a much smoother result by moving the Camera each frame in ``_process``, and following an interpolated target manually.
+
+Manual Camera interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Ensure Camera is using global space**
```

Comment:
```suggestion
**Ensure the Camera is using global coordinate space instead of the default local space.**
```

Reply:
Omitted the "instead of default local space", because local space can still be the same as global space (if the parent is identity or it is set as top level).